impeller/playground/backend/vulkan/playground_impl_vk.cc

```diff
@@ -88,6 +88,9 @@ PlaygroundImplVK::PlaygroundImplVK(PlaygroundSwitches switches)
   context_settings.shader_libraries_data = ShaderLibraryMappingsForPlayground();
   context_settings.cache_directory = fml::paths::GetCachesDirectory();
   context_settings.enable_validation = switches_.enable_vulkan_validation;
+  context_settings.fatal_missing_validations =
```

Comment:
IIRC both goldens and regular playground tests go through this path to create the context.

Reply:
I'm not sure.  I'd set a breakpoint in the impeller_golden_tests here to make sure.  We swap out playground tests for golden tests in that target so this might not be getting hit.  That's why we should make sure we have the test for that executable too (as mentioned in the other comment).

Reply:
Both impeller unittests and goldens have failed on the commit where I broke run_tests.py:

https://github.com/flutter/engine/pull/51378/checks?check_run_id=22626102272

I think that means its working.

Reply:
No its in there: https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8753556488722177297/+/u/test:_Impeller-golden__dart_and_engine_tests_for_host_release/stdout

It did hit a flake during one of the reruns though.

Reply:
That failure is when executing `impeller_unittests`.  Let's GVC when you have a chance.  I think I can clear this up in under 5 minutes over gvc =)